### ISSUE TITLE
{INFO] Формат представления экспоненты

### ISSUE THREAD

**Issue by VictorVG** [FarPlugins/Calculator#10]
С точки зрения нашего удобства вариант записи _digit_E_знакdigit_  удобнее, но с точки зрения математики вариант _digit_E_digit_/_digit_E_-digit_ считается допустимым. Я встречал его в математических работах, хотя для чтения он менее удобен чем вариант со знаком экспоненты. По крайней мере когда я учился наши преподаватели и после мы на кафедре знак экспоненты никогда не опускали, что ускоряет чтение сложных формул "с листа" и одновременно снижает число сомнений в правильности трактовки прочитанного.

**Comment by ctapmex** [comment]
привет.
принято. 
для контекста, это про https://forum.farmanager.com/viewtopic.php?p=168557#p168557

**Comment by VictorVG** [comment]
Да, я там вчера и увидел вопрос. С точки зрения ошибок я бы не опускал знак для исключения ошибок распознования типа HEX или FLOAT, а с точки зрения хранения данных в памяти и регистрах ЦП оба варианта одинаковы т.к в регистрах АЛУ мантисса и экпонента всегда имеют бит знака. Так что я думаю, что есть смысл его показывать - пусть и найдутся желающие поворчать, но зато мы однозначно исключим ошибку разпознования формата записи при её чтении оператором ЭВМ с экрана.
